filament/backend/src/metal/MetalDriver.h

```diff
@@ -129,7 +129,9 @@ class MetalDriver final : public DriverBase {
             uint32_t offset, uint32_t minIndex, uint32_t maxIndex, uint32_t count);
 
     void finalizeSamplerGroup(MetalSamplerGroup* sg);
-    void enumerateBoundUniformBuffers(const std::function<void(const UniformBufferState&,
+    void enumerateBoundUniformBuffers(const std::function<void(const BufferState&,
+            MetalBuffer*, uint32_t)>& f);
+    void enumerateBoundSsbos(const std::function<void(const BufferState&,
```

Comment:
Should both method be replaced by one generic `enumerateBoundBuffers` that takes a `BufferObjectBinding` instead? I guess it's not relevant for "vertex" buffer, since they're not bound.